storage/series.go

```diff
@@ -217,8 +217,11 @@ type seriesToChunkEncoder struct {
 	Series
 }
 
-// TODO(bwplotka): Currently encoder will just naively build one chunk, without limit. Split it: https://github.com/prometheus/tsdb/issues/670
+const seriesToChunkEncoderSplit = 120
```

Comment:
hm, I wonder if we could use existing constant: https://github.com/prometheus/prometheus/blob/d614ae9ecf1c1f70ca81e11d600079c4a3f866e1/tsdb/head.go#L2334 🤔 We only need to make sure we don't do any cycling package link.

Reply:
I guess we could try to figure something out. As you suspected right now there will be a import cycle and we would need to move the constant elsewhere.
In the end it might not make such a big difference. I don't expect this change before Prometheus v3.0.